pkg/generators/readers.go

```diff
@@ -237,6 +237,47 @@ func (g *ReadersGenerator) generateHelpers() error {
 			}
 			return
 		}
+
+		// ParseInteger reads a string and parses it to integer,
+		// if an error ocurred it returns a non-nil error.
+		func ParseInteger(s string) (*int, error) {
+			parsedInt64, err := strconv.ParseInt(s, 10, 64)
```

Comment:
This could be just `parsed` in all functions.

Don't we need to import `strconv`?

Reply:
Seems to compile...